types/pixi/grids/GridLayer.d.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * A CanvasLayer responsible for drawing a square grid
+ */
+declare class GridLayer extends CanvasLayer {
+  /**
+   * The Grid container
+   * (default: `null`)
```

Comment:
For properties you can use the `@defaultValue` doc tag instead.
```suggestion
   * @defaultValue `null`
```